src/components/app.test.jsx

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
```

Comment:
If you had more time, what would be your strategy for testing this app? What sort of testing frameworks do you like to use, and what, to you, are the markers of a "good" test?

Reply:
I'll be as transparent as possible in this topic, I'm definitely still learning to write better and more efficient tests but the one I've been using the most is the react/testing-library. Definitely something I need to dive deeper in and write more but in terms of a "good" test, I'd say being able to write tests that can better error handle. 

For example, if an api doesn't return anything, a city is spelled wrong, or even one case I saw now running my this weather app, if no city is recognized in my suggestions list (right now it shows a blank dropdown 🤦🏻‍♂️) Those would be some tests I'd want to run.